File: libgomp/testsuite/libgomp.fortran/task-detach-11.f90
! { dg-do run }

! Test the detach clause when the task is undeferred.

program task_detach_11
  use omp_lib

  integer (kind=omp_event_handle_kind) :: detach_event

  !$omp task detach (detach_event)
    call omp_fulfill_event (detach_event)
  !$omp end task
end program

File: libgomp/testsuite/libgomp.c-c++-common/task-detach-11.c
/* { dg-do run } */

#include <omp.h>

/* Test the detach clause when the task is undeferred.  */

int main (void)
{
  omp_event_handle_t event;

  #pragma omp task detach (event)
    omp_fulfill_event (event);
}
